### PR TITLE
feat: World ID Section updates

### PR DESCRIPTION
**Pull request by 0xPenryn**
includes documentation of World ID Reset, its impacts, and associated suggestions for integrations

### CODE REVIEW

File: src/pages/further-reading/world-id-reset.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# World ID Reset
+
+World ID 2.0 allows a user to reset their World ID in case their World ID has been lost or stolen. The user reverifies at the Orb, and their old World ID is invalidated. The user is then issued a new World ID and this new identity is verified at the Orb.  A user will not be able to perform a World ID Reset more than once every 14 days.
+
+## Impact on Sign-In with World ID
```
Comment (pdtfh) [inline src/pages/further-reading/world-id-reset.mdx:5]:
```suggestion
## Impact on Sign In with World ID
```

File: src/pages/further-reading/world-id-reset.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Impact on Anonymous Actions
+
+When a user performs a World ID Reset, they are issued a new World ID and their old World ID is invalidated. This means that any anonymous actions performed by the user will no longer be associated with their new World ID.
+
+<Note type="danger">If your application uses Anonymous Actions for sybil-resistance, this means that a user will be able to perform the action again with a new Nullifier Hash after performing a World ID Reset.</Note>
+
+### Suggested action
+
+We recommend rotating the action used in your application regularly and limiting the time an action is valid.
```
[the remark this answers is not in the document]
Comment (pdtfh) [reply src/pages/further-reading/world-id-reset.mdx:25]: more generally let's also simplify the text here, being less verbose will help

File: src/pages/id/on-chain.mdx [hunk 3]
```diff
@@ -41,25 +40,45 @@ const { address } = useAddress() // get the user's wallet address
 
 The World ID Router contract is what you should use to verify proofs. It is deployed on multiple chains, and you can find the addresses for each chain in our contracts [Address Book](/reference/address-book).
 
-## `hashToField` helper function
+## `hashToField` Helper Function
```
Comment (pdtfh) [inline src/pages/id/on-chain.mdx:43]: @m1guelpf are we still exposing this from IDKit?

**Review by pdtfh — APPROVED** [review]

File: src/pages/id/on-chain.mdx (hunk 3 above)
Comment (0xPenryn) [reply src/pages/id/on-chain.mdx:43]: hashToField is exported in `@worldcoin/idkit-core/hashing`

**Pull request merged**